### PR TITLE
List Group

### PR DESCRIPTION
**Pull request by bwarner**

### CODE REVIEW

File: lib/list-group/list-group-item.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  listGroupItemWarning,
+  listGroupItemInfo,
+  listGroupItemLight,
+  listGroupItemDark,
+  listGroupItemAction,
+} from '@farvisionllc/emotion-bootstrap/dist/bootstrap';
+
+const colors = {
+  primary: listGroupItemPrimary,
+
```
Comment (hafabenny) [inline lib/list-group/list-group-item.js]: why adding spaces between properties?

File: lib/list-group/list-group-item.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  const classes = cx(className, disabled ? 'disabled' : false, listGroupItem);
+  // Prevent click event when disabled.
+  if (disabled) {
+    attributes.onClick = handleDisabledOnClick;
+  }
+  return (
+    <Tag
+      {...attributes}
+      className={`${classes} ${color ? colors[color] : ''} ${
+        activeProp ? active : ''
```
Comment (hafabenny) [inline lib/list-group/list-group-item.js]: is activeProp && active the same in these cases?

Comment (bwarner) [reply lib/list-group/list-group-item.js]: Yes, cx combines emotion generated classnames in a new class with a new classnames.

File: lib/list-group/list-group-item.js (hunk 1 above)
Comment (bwarner) [reply lib/list-group/list-group-item.js]: that was some copy and pasting, plus search and replacing. unintentional.

**Pull request merged**